decoder/vaapidecoder_mpeg2.cpp

```diff
@@ -134,7 +134,7 @@ VaapiDecoderMPEG2::DPB::insertPictureToReferences(const PicturePtr& picture)
 
 YamiStatus VaapiDecoderMPEG2::DPB::insertPicture(const PicturePtr& picture)
 {
-    INFO("insertPicture to DPB size %lu", m_referencePictures.size());
+    INFO("insertPicture to DPB size %lu", (unsigned long)m_referencePictures.size());
```

Comment:
you can use %zu. https://stackoverflow.com/questions/2524611/how-can-one-print-a-size-t-variable-portably-using-the-printf-family